### DISCUSSION TITLE
v5.3

### DISCUSSION THREAD

**Discussion by gumbarros** [ncalc/ncalc#360] [General]
* Improve SerializationVisitor performance by @Bykiev in https://github.com/ncalc/ncalc/pull/353
* Remove nullable disable by @Bykiev in https://github.com/ncalc/ncalc/pull/355
* Improve performance of math functions by @Bykiev in https://github.com/ncalc/ncalc/pull/356
* Lambdas - use BigDecimal in Pow function by @Bykiev in https://github.com/ncalc/ncalc/pull/357
* Round - use int32 as digits parameter type by @Bykiev in https://github.com/ncalc/ncalc/pull/359
* Lambdas - add support for DecimalAsDefault by @Bykiev in https://github.com/ncalc/ncalc/pull/358

@Bykiev thanks for the work at these PRs.

Can we create a v5.3 release or do you want do more things before? Should we update the icon (#205)?

**Comment by Bykiev** [comment]
I think we can publish, no any new ideas at the moment :) About the icon - yes, we can update if you wish.

**Comment by Bykiev** [comment]
Maybe for v5.3 we can add .net 9 in target frameworks?

**Reply by gumbarros** [reply]
Sure, and C# 13 for the entire proj. I think we can use something maybe.

**Reply by Bykiev** [reply]
Would you like to contribute?
